packages/live-status-gateway/src/topics/activePlaylist.ts

```diff
@@ -287,6 +289,18 @@ export class ActivePlaylistTopic
 				this._globalAdLibActions = globalAdLibActions
 				break
 			}
+			case AdLibsHandler.name: {
```

Comment:
I feel like it would be a good idea to be consistent with the other cases for now and use a fixed string.
I do agree that something like this is a good idea, but I think it might need to be done as an enum. Importing the other files like this could result in an import loop which can cause issues

Reply:
An import loop is something I have not considered. I hope eventually those names won't be needed. I'd like us to eventually get rid of those large switches that needs a type assertion in every case, and do it in a way that makes easy to make sure the necessary piping is set up, which with the current implementation in `liveStatusServer.ts` is not.

Reply:
I think a circular dependency is actually unlikely, because the data flow in the handlers and topics is designed to be unidirectional.